README.md

```diff
@@ -279,8 +278,7 @@ This function is called anytime the internal state changes. This can be useful
 if you're using downshift as a "controlled" component, where you manage some or
 all of the state (e.g. isOpen, selectedItem, highlightedIndex, etc) and then
 pass it as props, rather than letting downshift control all its state itself.
-The parameters both take the shape of internal state (`{highlightedIndex:
-number, inputValue: string, isOpen: boolean, selectedItem: any}`) but differ
+The parameters both take the shape of internal state (`{highlightedIndex: number, inputValue: string, isOpen: boolean, selectedItem: any}`) but differ
```

Comment:
I'm guessing this reformatting happened because you either didn't install the project dependencies or the recent prettier release has some bugs 🤔

Reply:
I tried to follow the instructions as best as I could. Not sure where that was introduced, but somewhere between install and doing the contribution script.

Reply:
No worries :smile: